src/validation.cpp

```diff
@@ -837,6 +837,10 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
         return state.Invalid(TxValidationResult::TX_INPUTS_NOT_STANDARD, "bad-txns-nonstandard-inputs");
     }
 
+    if (InscriptionBytes(tx, m_view) > m_pool.m_max_inscription_bytes.value_or(0)) {
```

Comment:
Am I missing something? This seems to set the default behaviour to forbid any inscription data. I think this should be: `if (m_pool.m_max_inscription_bytes && *m_pool.m_max_inscription_bytes < InscriptionBytes(tx, m_view)) { ... }`. That also ensures `InscriptionBytes` isn't evaluated at all if it's not needed.